playbooks/ceph.yml

```diff
@@ -83,10 +83,35 @@
   hosts: "{{ cifmw_ceph_target | default('computes')  }}"
   gather_facts: true
   become: true
-  vars:
-    ansible_ssh_private_key_file: "{{ lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY') }}"
-  roles:
-    - role: cifmw_block_device
+  tasks:
+    - name: set cifmw_num_osds_perhost
+    # By defualt 1 OSD is created per node in case of multinode.
+    # 3 OSDS will be created for single node env to accomodate
+    # more ceph resources and avoid PG errors.
+      ansible.builtin.set_fact:
+        cifmw_num_osds_perhost: |
+          {% if groups[cifmw_ceph_target | default('computes')] | length == 1 %}
```

Comment:
This fails in molecule tests - guess you'll have to fake the ansible inventory in order to inject a "computes" entry?

`The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'computes'. 'dict object' has no attribute 'computes'`

Reply:
Thanks , i will fix in them in the next patch.